js/data/FinancialTimes_Bram-van-osta/assets/js/app.js

```diff
@@ -0,0 +1,31 @@
+"use strict";
+
+import getContributors from './getContributors';
+import getCommits from './getCommits';
+import orderCommitsByDay from './orderCommitsByDay';
+import formatCommitDataForChart from './formatCommitDataForChart';
+import generateDonutChart from './generateDonutChart';
+
+document.addEventListener('DOMContentLoaded', function () {
+
+    // Start by getting all the contributors to find out the total number of commits
+    getContributors().then(function (contributors) {
+        var totalCommits = 0;
+
+        // Get the total number of commits
+        contributors.forEach(function (contributor) {
+            totalCommits += contributor.contributions;
+        });
```

Comment:
Other options to do this include using reduce:
```js
var totalCommits = contributors.reduce(function(acc, curr){
    return acc + contributor.contributions
}, 0)
```

or [lodash.sum](https://lodash.com/docs/4.16.6#sum) or any other package